group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -3438,15 +3477,30 @@ public void replay(
         String groupId = key.groupId();
         String memberId = key.memberId();
 
-        ConsumerGroup group = getOrMaybeCreatePersistedConsumerGroup(groupId, false);
-        ConsumerGroupMember oldMember = group.getOrMaybeCreateMember(memberId, false);
-
         if (value != null) {
+            ConsumerGroup group = getOrMaybeCreatePersistedConsumerGroup(groupId, true);
```

Comment:
not sure why we decide to create new group. If it does create a new group, the `getOrMaybeCreateMember` will throw `UnknownMemberIdException` subsequently.

Reply:
Maybe we want the member to also be true?

Reply:
Indeed. It should be true. I will fix that.